app/src/controllers/ChannelController.js

```diff
@@ -0,0 +1,117 @@
+const log = require('loglevel')
+const ObservableStore = require('obs-store')
+
```

Comment:
pls use import syntax

Reply:
Opss that was unintented ported from the old codebase. Fixed now!